internal/feather.go

```diff
@@ -65,7 +109,8 @@ func inputsToArrowBytes(inputs map[string]any) ([]byte, error) {
 		if !ok {
 			return nil, fmt.Errorf("failed to find input values for feature '%s'", field.Name)
 		}
-		reflectKind := reflect.ValueOf(values).Type().Elem().Kind()
+		elem := reflect.ValueOf(values).Type().Elem()
```

Comment:
nit: you can abbreviate `reflect.ValueOf(x).Type()` as `reflect.TypeOf(x)`. But also, it's probably best to confirm that `values` is really a slice type or you'll get a very confusing error later. Something like

```go
valuesType := reflect.TypeOf(values)
if valuesType.Kind() != reflect.Slice {
  return fmt.Errorf("the value provided for feature '%s' is not of slice type; it has type %v", field.Name, valuesType)
}
elemType := valuesType.Elem()
```